controls/radmap/features/providers/azuremapprovider.md

```diff
@@ -0,0 +1,143 @@
+    <telerik:RadMap x:Name="map">
+        <telerik:RadMap.Providers>
+            <telerik:AzureMapProvider SubscriptionKey="Azure_Maps_Subscription_Key"
+                                      TileSet="DarkGrey"/>
+        </telerik:RadMap.Providers>
+    </telerik:RadMap>
+{{endregion}}
+
+__AzureMapProvider with specified tileset__
+
+![{{ site.framework_name }} AzureMapProvider with specified tileset](images/radmap-features-providers-azuremapprovider-1.png)
+
+By default RadMap's navigation UI doesn't display any modes for the AzureMapProvider. To enable this, set the provider's `UserAccessibleModes` collection property.
+
+#### __[XAML] Setting the tileset modes__
+{{region radmap-features-providers-azuremapprovider-2}}
+    <telerik:RadMap>
+        <telerik:RadMap.Providers>
+            <telerik:AzureMapProvider SubscriptionKey="Azure_Maps_Subscription_Key"
+                                      TileSet="DarkGrey">
+                <telerik:AzureMapProvider.UserAccessibleModes>
+                    <telerik:AzureTileSet>DarkGrey</telerik:AzureTileSet>
+                    <telerik:AzureTileSet>HybridDarkGrey</telerik:AzureTileSet>
+                </telerik:AzureMapProvider.UserAccessibleModes>
+            </telerik:RadMap.Providers>
+        </telerik:AzureMapProvider>
+    </telerik:RadMap>
+{{endregion}}
+
+__AzureMapProvider with specified AzureTileSet instances for the UserAccessibleModes property__
+
+![{{ site.framework_name }} AzureMapProvider with specified AzureTileSet instances for the UserAccessibleModes property](images/radmap-features-providers-azuremapprovider-2.png)
+
+## Styling Vector Tiles via Custom Styles
+
+The RadMap's default styling of vector tiles does not support the vector tiles of the Azure Maps services. Instead, you can use a third-party styles to style them, for example, using [TomTom Map Styles](https://developer.tomtom.com/map-display-api/documentation/mapstyles/map-styles). After retrieving the custom style, set it to the `StyleFileSource` property of the AzureMapProvider instance.
+
+## Manually Customizing the Appearance of the Vector Tiles via NoStyleVectorTileRenderingEventArgs Event
+
+An alternative approach for styling the vector tiles is via the `NoStyleVectorTileRenderingEventArgs` event. It will be raised for each vector tile before being rendered.
+
+The event arguments of the NoStyleVectorTileRendering event are of the type `NoStyleVectorTileRenderingEventArgs` and expose the following properties:
+
+* `ZoomLevel`&mdash;The ZoomLevel property provides the ability to control the zoom level of the vector tile.
+* `BackgroundColor`&mdash;This property allows you to specify the background color of the vector tile.
+* `Layers`&mdash;Collection that contains `VectorTileLayerRenderInfo` instances.
+
+>important The NoStyleVectorTileRendering event will be raised when the `IgnoreStyle` property of the AzureMapProvider instance is set to `False`.
```

Comment:
Shouldn't the "NoStyleVectorTileRendering" be quoted like this `NoStyleVectorTileRendering`?